stanford/bin/certs-generate-course

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+_required_variables='REMOTE_BOX COURSE_ID SYSTEM_USER'
+for _variable in ${_required_variables}; do
+    test -n "${!_variable}" || { echo "$_variable must be set"; exit 1; }
+done
+
+# optional variables
```

Comment:
Normally, I'm not a huge fan of comments like this, but this `bash` syntax is opaque enough that I think it warrants it.